packages/core/src/common/predicates.ts

```diff
@@ -40,12 +40,11 @@ export const isValid =
 export const isSupportedDataType =
   (supportsString: boolean) =>
   ({ dataType }: { dataType: DataType }) =>
-    (supportsString && dataType === DataType.STRING) || dataType !== DataType.STRING;
+    (supportsString && dataType === 'STRING') || dataType !== 'STRING';
```

Comment:
Why is this not using the constant?

Reply:
Refactored to utilize `DATA_TYPE` contsant